<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="0" column="2" colspan="4">
        <widget class="QCheckBox" name="chkPrim">
         <property name="whatsThis">
          <string>В справочнике МКБ обычно отмечается звёздочкой</string>
         </property>
         <property name="text">
          <string>уточняется в других рубриках</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QTextEdit" name="edtName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cу&amp;бклассификация
МКБ по 5 знаку</string>
         </property>
         <property name="buddy">
          <cstring>cmbMKBSubclass</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="5">
        <widget class="CRBComboBox" name="cmbMKBSubclass">
         <property name="whatsThis">
          <string>Последующее уточнение кода диагноза, не предписанное справочником МКБ</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblExSubclass">
         <property name="text">
          <string>Расширенная субклассификация</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="5">
        <widget class="CMKBExSubclassInDocTableView" name="tblExSubclass"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblCharacters">
         <property name="text">
          <string>&amp;Характер</string>
         </property>
         <property name="buddy">
          <cstring>cmbCharacters</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="5">
        <widget class="QComboBox" name="cmbCharacters">
         <property name="whatsThis">
          <string>Допустимый характер заболевания, применяется для контроля ввода</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblEndDate">
         <property name="text">
          <string>Дата окончания применения</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="CDateEdit" name="edtEndDate">
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>122</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblDuration">
         <property name="text">
          <string>&amp;Длительность</string>
         </property>
         <property name="buddy">
          <cstring>edtDuration</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="edtDuration">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="4">
        <spacer name="spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>223</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblSex">
         <property name="text">
          <string>&amp;Пол</string>
         </property>
         <property name="buddy">
          <cstring>cmbSex</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="cmbSex">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ж</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="2" colspan="4">
        <spacer name="spacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>223</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lblAge">
         <property name="text">
          <string>Возраст</string>
         </property>
         <property name="buddy">
          <cstring>cmbBegAgeUnit</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QComboBox" name="cmbBegAgeUnit">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Д</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Н</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>М</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Г</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtBegAgeCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="inputMask">
            <string>000; </string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblAgeSep">
           <property name="text">
            <string>по</string>
           </property>
           <property name="buddy">
            <cstring>cmbEndAgeUnit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbEndAgeUnit">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Д</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Н</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>М</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Г</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtEndAgeCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="inputMask">
            <string>000; </string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="5">
        <spacer name="spacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>86</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lblService">
         <property name="text">
          <string>&amp;Услуга</string>
         </property>
         <property name="buddy">
          <cstring>cmbService</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="3">
        <widget class="CRBComboBox" name="cmbService"/>
       </item>
       <item row="9" column="4" colspan="2">
        <widget class="QToolButton" name="btnSelectService">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lblRequiresFillingDispanser">
         <property name="text">
          <string>Требует заполнения ДН</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="5">
        <widget class="QComboBox" name="cmblRequiresFillingDispanser">
         <item>
          <property name="text">
           <string>Никогда</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Иногда</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Всегда</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>Идентификация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CMKBExSubclassInDocTableView</class>
   <extends>QTableView</extends>
   <header>RefBooks/MKB/MKBExSubclassInDocTableView.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
